src/steps/findings.ts

```diff
@@ -0,0 +1,176 @@
+import {
+  createDirectRelationship,
+  createIntegrationEntity,
+  IntegrationStep,
+  IntegrationStepExecutionContext,
+  RelationshipClass,
+  IntegrationMissingKeyError,
+} from '@jupiterone/integration-sdk-core';
+
+import { createAPIClient } from '../client';
+import { IntegrationConfig } from '../types';
+import { getVulnerabilityLink, getVulnerabilityNumber } from '../util';
+
+export async function fetchFindings({
+  instance,
+  jobState,
+}: IntegrationStepExecutionContext<IntegrationConfig>) {
+  const apiClient = createAPIClient(instance.config);
+
+  await apiClient.iterateFindings(async (finding) => {
+    const findingProps = finding.resource;
+    const findingEntity = await jobState.addEntity(
+      createIntegrationEntity({
+        entityData: {
+          source: finding,
+          assign: {
+            _type: 'cobalt_finding',
+            _class: 'Finding',
+            _key: findingProps.id,
+            tag: findingProps.tag,
+            name: findingProps.title,
+            displayName: findingProps.title,
+            description: findingProps.description,
+            category: 'Penetration Testing',
+            typeCategory: findingProps.type_category,
```

Comment:
Is this something that could be stored in `category`?

Reply:
Yep! Done